react/SearchWrapper.js

```diff
@@ -1,63 +1,64 @@
 import PropTypes from 'prop-types'
-import React, { Component } from 'react'
-import { Helmet, withRuntimeContext } from 'vtex.render-runtime'
+import React, { Fragment, useMemo } from 'react'
+import { Helmet, useRuntime } from 'vtex.render-runtime'
 
-import DataLayerApolloWrapper from './components/DataLayerApolloWrapper'
 import { capitalize } from './utils/capitalize'
+import useDataPixel from './hooks/useDataPixel'
+
 const APP_LOCATOR = 'vtex.store'
 
-class SearchWrapper extends Component {
-  static propTypes = {
-    /** Route parameters */
-    params: PropTypes.shape({
-      category: PropTypes.string,
-      department: PropTypes.string,
-      term: PropTypes.string,
-    }),
-    /** Render runtime context */
-    runtime: PropTypes.shape({
-      page: PropTypes.string.isRequired,
-      account: PropTypes.any,
-      getSettings: PropTypes.func,
-    }),
-    /** Search query result */
-    searchQuery: PropTypes.object.isRequired,
-    /** Component to be rendered */
-    children: PropTypes.node.isRequired,
+const pageCategory = (products, params) => {
```

Comment:
Why not a regular function? First things first, since it's a component file I think the most important here is the component itself, so u could move these aux functions to the bottom of the file.

Reply:
if I move them to the bottom the eslint will complain saying that I'm using the function before it's defined.